zebra-utils/src/bin/zebra-tip-height/args.rs

```diff
@@ -0,0 +1,24 @@
+//! zebra-tip-height arguments
+//!
+//! For usage please refer to the program help: `zebra-tip-height --help`
+
+#![deny(missing_docs)]
+#![allow(clippy::try_err)]
```

Comment:
This lint should be allowed by default now.

Reply:
Fixed.